File: APP/app/src/main/java/com/TextbookAnswers/Answer.java
package com.textbookanswers;

public class Answer {

    public int id,questionId;

    public int number;
    public String question,answer,user;
    public int upvotes,downvotes;

    public Answer(int id,int questionId,String answer, int upvotes, int downvotes) {
        this.id = id;
        this.questionId = questionId;
        this.question = Repository.getExercise(questionId).question;
        this.answer = answer;
        this.upvotes = upvotes;
        this.downvotes = downvotes;
    }

    public Answer(String question, String answer, int upvotes, int downvotes) {
        this.question = question;
        this.answer = answer;
        this.upvotes = upvotes;
        this.downvotes = downvotes;
    }
}

File: API/Domain/Entities/Answer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Answer
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public int Upvotes { get; set; }
        public int Downvotes { get; set; }

        public int Score => Upvotes - Downvotes;

        public int QuestionId { get; set; }
        public Question Question { get; set; }

        public int AuthorId { get; set; }
        public User Author { get; set; }
    }
}
